include/wlr/types/wlr_layer_shell_v1.h

```diff
@@ -43,16 +43,27 @@ struct wlr_layer_shell_v1 {
 	void *data;
 };
 
+enum wlr_layer_surface_v1_state_field {
+	WLR_LAYER_SURFACE_V1_STATE_ANCHOR = 1 << 0,
+	WLR_LAYER_SURFACE_V1_STATE_EXCLUSIVE_ZONE = 1 << 1,
+	WLR_LAYER_SURFACE_V1_STATE_MARGIN = 1 << 2,
+	WLR_LAYER_SURFACE_V1_STATE_KEYBOARD_INTERACTIVITY = 1 << 2,
```

Comment:
A typo: `WLR_LAYER_SURFACE_V1_STATE_MARGIN` and `WLR_LAYER_SURFACE_V1_STATE_KEYBOARD_INTERACTIVITY` shouldn't be equal.